src/database/migrations/20191126082619-create-volunteer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.createTable("volunteers", {
```

Comment:
Let's use "Volunteers" for the table name (with a capital "V"). It's just a convention thing.